fdbserver/include/fdbserver/BlobGranuleServerCommon.actor.h

```diff
@@ -38,6 +39,8 @@
 
 #include "flow/actorcompiler.h" // has to be last include
 
+#define BW_DEBUG false
```

Comment:
Once the implementation is left in BlobWoker.cpp, then this #define need not to move.

Reply:
Also, BW == BlobWorker, it probably should be placed in BlobWokerCommon or BlobWorker file.